styles.css

```diff
@@ -0,0 +1,214 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Domine:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 125vh;
+  background: #1c1a19;
+  background-size: cover;
+}
+
+.portfolio {
+  width: 100%;
+  height: 125vh;
+  left: 0;
+  top: 0;
+  margin: 0 auto;
+}
+
+.details {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  height: auto;
+  left: 0;
+  top: 0;
+  background: #3e3c3c;
+  flex-wrap: wrap;
+}
+
+.inside-name {
+  width: 92px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.section-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 10px 0;
+  gap: 12px;
+}
+
+.greeting {
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 82px;
+  color: #f55800;
+  margin: 24px 0 24px 0;
+}
+
+.introduction {
+  width: 566px;
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin: 24px 0 24px 0;
+}
+
+.paragraph {
+  width: 566px;
+  height: 123px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+  margin: 24px 0 24px 0;
+}
+
+.socials-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  position: absolute;
+  margin: 270px 0 0 30px;
+  order: 0;
+}
+
+.socials-container a {
+  margin: 20px 0 20px 0;
+}
+
+@media all and (max-width: 375px) {
+  .details {
+    background: #3e3c3c;
+    width: 375px;
+    height: 70px;
+  }
+
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: auto;
+    background: #1c1a19;
+    background-size: cover;
+  }
+
+  .section-container {
+    width: 375px;
+    height: 500px;
+    left: 0;
+    top: 92px;
+    right: 0;
+    bottom: 0;
+    flex-basis: 100%;
+    background-image: url("images/Group.png");
+    background-repeat: no-repeat;
+    background-position: 240px 200px;
+  }
```

Comment:
- [OPTIONAL] Kindly make sure to provide enough space between the contents in the headline section.
- [OPTIONAL] I highly recommend to double check the font size of your texts and the size of the icon and making sure it looks as much as possible as the provided Figma design.

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/213654359-3f95cce1-abc8-4093-aff9-5e4994b0b8aa.png)

   - ### FIGMA DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/213654498-f289f234-77cc-44fb-9240-92c07830b9ac.png)